recipes/requests-ftp/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{%set name = "requests-ftp" %}
+{%set version = "0.3.1" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "7504ceb5cba8a5c0135ed738596820a78c5f2be92d79b29f96ba99b183d8057a" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - requests >=2.3.0
+    - wsgiref  # [py2k]
+
+test:
+  imports:
+    - requests_ftp
+
+about:
+  home: https://github.com/Lukasa/requests-ftp
+  license: Apache-2
+  license_file: LICENSE
+  license_family: Apache
+  summary: 'An FTP transport adapter for use with the Python Requests library'
+  dev_url: https://github.com/Lukasa/requests-ftp
+
+extra:
+  recipe-maintainers:
+    - mvanniekerk
```

Comment:
Oops, guessing this was suppose to be you, @mariusvniekerk. Though we should have caught this too. Do you want to send a PR to the feedstock to correct this? Would be happy to merge if you ping me.